src/olympia/reviewers/utils.py

```diff
@@ -1139,6 +1139,14 @@ def approve_latest_version(self):
         subject = u'Mozilla Add-ons: %s %s signed and ready to download'
         self.log_action(amo.LOG.APPROVE_VERSION)
 
+        if self.human_review:
+            if self.version.needs_human_review:
+                self.version.update(needs_human_review=False)
+
+            VersionReviewerFlags.objects.filter(
+                version=self.version
+            ).update(needs_human_review_by_mad=False)
+
         self.notify_email(template, subject, perm_setting=None)
 
         log.info(u'Making %s files %s public' %
```

Comment:
Might be a different issue but in `block_multiple_versions`, shouldn't we also clear the `VersionReviewerFlags`?

Reply:
Oh yeah I was wondering about that and forgot to ask. @wagnerand , should we clear human review/human review by mad flags in the block multiple versions action as well ?

Reply:
I think it makes sense to clear the flags in that case, yes.

Reply:
I added https://github.com/mozilla/addons-server/pull/15044/commits/d221ea9855805e96ab1cb7f6b4dee9a6b699d28e which clears the flag in `block_multiple_versions` action but also refactors things so that we have centralized methods to clear the needs human review flags.